pegasusio/zarr_utils.py

```diff
@@ -303,10 +313,16 @@ def write_dataframe(self, parent: zarr.Group, name: str, df: pd.DataFrame) -> No
         attrs_dict['index_name'] = df.index.name if df.index.name is not None else 'index'
         self.write_series(group, '_index', df.index.values)
         for col in df.columns:
-            self.write_series(group, col, df[col].values)
+            if isinstance(df[col].values[0], np.ndarray):
```

Comment:
Change to checking if a `PIL.Image` object.

Reply:
check if i have a pointer on the png image, as long as we dont start processing the pil image, it is just a pointer